factory/glideFactoryEntryGroup.py

```diff
@@ -29,7 +29,7 @@
 from glideinwms.factory import glideFactoryLib as gfl
 from glideinwms.factory import glideFactoryPidLib
 from glideinwms.lib import classadSupport, cleanupSupport, logSupport
-from glideinwms.lib.fork import fetch_fork_result_list, ForkManager
+from glideinwms.lib.fork import fetch_fork_result_list, ForkManager, print_child_processes
```

Comment:
Isn't this a syntax error? I mean the comma after ForkManager?

Reply:
I don't think. And pylint and the unit test are OK. The comma should be after all items after the first one: https://docs.python.org/3/reference/simple_stmts.html#the-import-statement

Reply:
For some reason I saw `ForkManager` written in red and my brain registered it as the`import keyword`. all good.